### PR TITLE
Multi sensor loading scaling file

### PR DESCRIPTION
**Pull request by armando-montalvo**
I modified the file taking into account the suggestions of @toliver and @dogoepp.

### CODE REVIEW

File: optoforce/src/optoforce/optoforce.py [hunk 1]
```diff
@@ -91,6 +88,17 @@ def __init__(self):
...
+        self._scale = rospy.get_param("~scale")
+
+        if len(self._scale) != self._nb_sensors:
+            raise ValueError("The number of sensors and scale factor vectors given doesn't match.") 
+        else:
+            for x in range(self._nb_sensors):
+                if len(self._scale[x]) != self._nb_axis:
+                    print(self._nb_axis)
+                    print(self._scale[x])
+                    raise ValueError("Number of axis and scaling factors given doesn't match.")
```
Comment (dogoepp) [inline optoforce/src/optoforce/optoforce.py]: Maybe you can concatenate the numbers in the Exception's message instead of printing them ?

Comment (armando-montalvo) [inline optoforce/src/optoforce/optoforce.py]: Oh, those prints were actually troubleshooting, sorry, but I could print the sizes in the error.

File: optoforce/config/SR_OMD45FE1000N.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+counts_n_fx: 71.07
```
Comment (toliver) [inline optoforce/config/SR_OMD45FE1000N.yaml]: The contents of the file should be modified to:

``` yaml
scale:
   - [71.07, 69.00, 6.11, 1.00, 1.00, 1.00]
```

Comment (toliver) [inline optoforce/config/SR_OMD45FE1000N.yaml]: And maybe this file should be renamed to something generic before we can merge it. Something like `single_channel_6_axis_generic_scale.yaml`.

Comment (toliver) [inline optoforce/config/SR_OMD45FE1000N.yaml]: Another one called `multi_channel_3_axis_generic_scale.yaml` could be added, as the default value in hte launchfile is "m-ch/3-axis".

File: optoforce/launch/optoforce.launch [hunk 3]
```diff
@@ -31,7 +31,8 @@
   <!-- Numerical index of the first sensor. The rest (if this is a multi-channel DAC)
   will get the consecutive numbers -->
   <arg name="starting_index" default="0"/>
-
+    <!-- Direction of scaling file -->
+  <arg name="scaling_file" default=""/>
```
Comment (toliver) [inline optoforce/launch/optoforce.launch]: And a default value could be defined here:

``` xml
<arg name="scaling_file" default="$(find optoforce)/config/multi_channel_3_axis_generic_scale.yaml"/>
```

**Comment by dogoepp** [comment]
Any News on this pull request ?

**Comment by armando-montalvo** [comment]
Well, I've just made the changes and checked twice if there was something wrong, but I think it's good. Thanks a lot @toliver and @dogoepp for the feedback!

**Comment by dogoepp** [comment]
Thanks a lot @armando-montalvo ! I hope it can be merged soon.

**Comment by armando-montalvo** [comment]
@toliver, I was checking how to contact you directly, but as there is no e-mail listed in your profile, I have to ask you here. I am an intern in Fraunhofer IPA and was assigned to help maintain this code. We would like to add a wiki page for this package in the ROS wiki and modify the code to meet ROS Industrial standards, if that is ok for you guys. You can contact me directly in my e-mail (It's in my profile). Thanks!

**Comment by toliver** [comment]
@armando-montalvo I've just replied via email.

**Comment by toliver** [comment]
@armando-montalvo there seems to be some conflicts. Can you solve them so that we can merge?

**Comment by toliver** [comment]
And indeed thanks for your collaboration!!!

**Pull request merged**